scripts/create-release

```diff
@@ -21,6 +21,9 @@ fi
 
 PR="$1"; shift
 
+# Allow just pasting the PR URL
+PR=$(echo "$PR" | sed -E 's#https://github.com/18F/identity-idp/pull/([0-9]+).*#\1#')
```

Comment:
Question: Should we incorporate the `GH_REPO` value in this string?

Reply:
probably!